### PR TITLE
Updates to Team Templates Documentation

### PR DESCRIPTION
**Pull request by phecdalouie**
- Added 'operation' as a relationship type in team
- Added education specific team specializations

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[16fcce1](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/16fcce14deaf3956fe2fb105b98aa50a8716ab67)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[16fcce1](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/16fcce14deaf3956fe2fb105b98aa50a8716ab67)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[16fcce1](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/16fcce14deaf3956fe2fb105b98aa50a8716ab67)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901181910496555-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Od4qNrWR126Z1LuwhcC4ka13b%2BoE2zbtYMHNGy3jBm4%3D&st=2019-01-18T19%3A08%3A59Z&se=2019-02-18T19%3A13%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/teamspecialization.md [hunk 1]
```diff
@@ -17,4 +17,7 @@ Indicates whether the [team](../resources/team.md) is intended for a particular
 | Member             | Value | Description                                                                |
 | :----------------- | :---- | :------------------------------------------------------------------------- |
 | none               | 0     | Default type for a team which gives the standard team experience.          |
-| unknownFutureValue | 7     | Sentinel value reserved as a placeholder for future expansion of the enum. |
+| educationStandard  | 1     | Team created by an education user. All teams created by education user are of type Edu. |
+| educationClass     | 2     | Team experience optimized for a class. This enables segmentation of features across O365. |
+| educationProfessionalLearningCommunity | 3 | Team experience optimized for a PLC. Learn more about PLC at https://en.wikipedia.org/wiki/Professional_learning_community |
```
Comment (nkramer) [inline api-reference/beta/resources/teamspecialization.md]: Add a real hyperlink -- [text](url)

And is a link in Wikipedia the best we can do?

File: api-reference/beta/resources/teamspecialization.md [hunk 2]
```diff
@@ -17,4 +17,7 @@ Indicates whether the [team](../resources/team.md) is intended for a particular
...
+| educationStaff     | 4     |  Team type for optimized experience for staff in an institution. Where staff leader, like principal, is admin and teachers are members in a team which comes with a specialized notebook. Learn more the staff notebook at https://www.onenote.com/staffnotebookedu |
```
Comment (nkramer) [inline api-reference/beta/resources/teamspecialization.md]: Another link

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b93e2c2](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/b93e2c23d16f6c33e2b37217b8090a23a2ab8f1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901181928132873-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=HG9RA9qHzkeddMl6iMa1LUXa4NDcTRcCSXHx6ZIOhA8%3D&st=2019-01-18T19%3A26%3A32Z&se=2019-02-18T19%3A31%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[532da5d](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/532da5ddb9dd276ebf94c9017e9bbc781abe4866)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[532da5d](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/532da5ddb9dd276ebf94c9017e9bbc781abe4866)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

File: api-reference/beta/resources/teamspecialization.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/resources/teamspecialization.md]: Yes, please don't link to Wikipedia in our docs. Can you link to something like this? http://www.allthingsplc.info/about

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[532da5d](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/532da5ddb9dd276ebf94c9017e9bbc781abe4866)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901181929043890-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2rdbsUwzOVcQjE3XJdtOcE8Vq3bdX461R4NNWc%2FzYjc%3D&st=2019-01-18T19%3A29%3A48Z&se=2019-02-18T19%3A34%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97c6e12](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/97c6e126a70e7c8048f8926c07e3815542936453)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97c6e12](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/97c6e126a70e7c8048f8926c07e3815542936453)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97c6e12](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/97c6e126a70e7c8048f8926c07e3815542936453)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901182208039606-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EdFLdpQE1wFh63jtArdO61Ul5wel7vyT4S63rLy29go%3D&st=2019-01-18T22%3A06%3A26Z&se=2019-02-18T22%3A11%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[624f7e5](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/624f7e56022a8f0a74470282ff9675a0381b46d8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901182228225772-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=INiw9zfEMJuoXS32wWh4GZWRQuDaiyXVCgNvxz4xDY0%3D&st=2019-01-18T22%3A26%3A39Z&se=2019-02-18T22%3A31%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d665605](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/d6656053e310287bf46e8c39812c4e977bb4a9a6)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d665605](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/d6656053e310287bf46e8c39812c4e977bb4a9a6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901252348263013-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5H9b9oB03sHGVc7WwILISxVmnWFOC5uj4STtIOUfUtE%3D&st=2019-01-25T23%3A46%3A58Z&se=2019-02-25T23%3A51%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a1d6cc2](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/a1d6cc24003108dbf1aa3bd39824b695e027e468)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a1d6cc2](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/a1d6cc24003108dbf1aa3bd39824b695e027e468)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a1d6cc2](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/a1d6cc24003108dbf1aa3bd39824b695e027e468)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901252349162955-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PK6johD8t0wkdg0XVrwM%2Fpexc%2BHjjNLXONatbS2ERV4%3D&st=2019-01-25T23%3A50%3A35Z&se=2019-02-25T23%3A55%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f3b8195](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/f3b81955380a14764197806d7b800ce587bb0609)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f3b8195](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/f3b81955380a14764197806d7b800ce587bb0609)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9f80e1f](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/9f80e1f4d597dd2aa05aa81a7160ba6fbd7eb90d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9f80e1f](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/9f80e1f4d597dd2aa05aa81a7160ba6fbd7eb90d)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[9f80e1f](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/9f80e1f4d597dd2aa05aa81a7160ba6fbd7eb90d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/team-post.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/api/team-post.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901260037366164-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LZ%2FKzqmojgYpM2%2BzAl3o%2Bf0EX722p0c0oN%2B6LQSlEUs%3D&st=2019-01-26T00%3A36%3A17Z&se=2019-02-26T00%3A41%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[0be7f9b](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/0be7f9b2cae031ac225a4d6fde3b6e8a85cc0967)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[0be7f9b](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/0be7f9b2cae031ac225a4d6fde3b6e8a85cc0967)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[0be7f9b](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/0be7f9b2cae031ac225a4d6fde3b6e8a85cc0967)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/team-post.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/api/team-post.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901260100115088-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RiCQswsrRdniR%2Bt1YrxotnW6UZxHbbe7ivocWNHAUXc%3D&st=2019-01-26T00%3A58%3A44Z&se=2019-02-26T01%3A03%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3f199a9](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/3f199a94e9713225957f15e4cf5d9904b9b29e97)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3f199a9](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/3f199a94e9713225957f15e4cf5d9904b9b29e97)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3f199a9](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/3f199a94e9713225957f15e4cf5d9904b9b29e97)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/team-post.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/api/team-post.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901260113370459-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=O4Dd5MHfdXAhCIjuYHMTE5MpMpcCs4TF5qHWcVRGBNE%3D&st=2019-01-26T01%3A12%3A06Z&se=2019-02-26T01%3A17%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[7d68545](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/7d68545a9e6a2df876f52a9625151d9335a78365)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fb0a84e](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/fb0a84e5cda145a7b794fa85c936870fb8c6e73e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fb0a84e](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/fb0a84e5cda145a7b794fa85c936870fb8c6e73e)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[fb0a84e](https://github.com/phecdalouie/microsoft-graph-docs/pull/3819/commits/fb0a84e5cda145a7b794fa85c936870fb8c6e73e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/team-post.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/api/team-post.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/team.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/team.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/teamspecialization.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/api-reference/beta/resources/teamspecialization.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/phecdalouie/microsoft-graph-docs/blob/phlouie-updateTemplatesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C26%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901260131169397-3819%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=K89RSVQ5R5ssr%2FP%2B2VhyeCFIRZAuiSIRQV4a2tw%2Ftmk%3D&st=2019-01-26T01%3A30%3A05Z&se=2019-02-26T01%3A35%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**